src/patterns/definitions.owl

```diff
@@ -7,8 +7,142 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033001> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033001> "blood vessel endothelial cell of arteriole of lymph node"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033001> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000071> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410042>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033002> (neuroendocrine cell of epithelium of crypt of Lieberkuhn)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:31390635"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033002> "A(n) neuroendocrine cell that is part of a(n) epithelium of crypt of Lieberkuhn."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033002> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033002> "neuroendocrine cell of epithelium of crypt of Lieberkuhn"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033002> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000165> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_0011184>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033003> (myoepithelial cell of bronchus submucosal gland)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:19996346"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:5487122"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033003> "A(n) myoepithelial cell that is part of a(n) bronchus submucosal gland."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> <http://purl.obolibrary.org/obo/CL_4033003> <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/CL_4033003> "myoepithelial cell of bronchus submucosal gland"^^xsd:string)
+EquivalentClasses(<http://purl.obolibrary.org/obo/CL_4033003> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/CL_0000185> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/UBERON_8410043>)))
+
+# Class: <http://purl.obolibrary.org/obo/CL_4033004> (smooth muscle cell of taenia coli)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:10715339"^^xsd:string) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:22320805"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/CL_4033004> "A(n) smooth muscle cell that is part of a(n) taenia coli."^^xsd:string)
```

Comment:
PMID:10715339 seems to be behind a paywall and investigates pathological states. No mention of the cell type in abstract. 
PMID: [22320805](https://pubmed.ncbi.nlm.nih.gov/22320805) seems OK, although it describes gross radiographic studies and nothing at the cellular level.

Reply:
Considering adding variant spellings of taenia (e.g., tenia) as synonyms.